app/views/shared/authentication/_forem_creator_signup.html.erb

```diff
@@ -1,3 +1,94 @@
-<div class="mx-auto mt-8 mb-10 p-4 fs-2xl lh-base align-center" style="width:96%;max-width:1300px;">
-  <%= render "shared/authentication/email_registration_form" %>
-</div>
+<%= javascript_packs_with_chunks_tag "foremCreatorSignup", defer: true %>
+<main id="main-content">
+  <div class="mx-auto mt-8 mb-10 p-4 fs-2xl lh-base align-center" style="width:96%;max-width:1300px;">
+    <div id="sign-in-password-form" class="mt-8 mb-6 crayons-card p-7 align-left mx-auto" style="max-width:580px;">
+      <% if flash[:notice] %>
+        <div class="crayons-notice crayons-notice--danger mb-6" role="alert">
+          <%= flash[:notice] %>
+        </div>
+      <% end %>
+
+      <% any_errors = defined?(resource) && resource&.errors&.any? %>
+
+      <%= form_for(@user, as: :user, data: { testid: "creator-signup-form" }, url: registration_path(:user)) do |f| %>
+        <% if any_errors %>
+          <div class="crayons-card crayons-card--secondary crayons-notice crayons-notice--danger my-8" role="alert" data-testid="signup-errors">
+            <div class="crayons-card__header">
+              <h3 class="crayons-card__headline">
+                Whoops, we found <%= pluralize(resource.errors.size, "problem") %>
+              </h3>
+            </div>
+            <div class="crayons-card__body">
+              <ul>
+                <% resource.errors.full_messages.each do |message| %>
+                  <li><%= message %></li>
+                <% end %>
+              </ul>
+            </div>
+          </div>
+        <% end %>
+
+        <div class="align-center">
+          <p class="pb-4 fs-3xl fw-bold">Let's start your Forem journey!</p>
+          <p class="color-base-70 fs-xl">Create your admin account first.</p>
+          <p class="color-base-70 fs-xl">Then we'll walk you through your Forem setup.</p>
+        </div>
+
+        <div class="crayons-field mt-6">
+          <%= f.label :name, "Name", for: "name", class: "crayons-field__label" %>
+          <%= f.text_field :name, id: "name", placeholder: "John Doe, john_doe, john, ...", class: "crayons-textfield js-creator-signup-name", required: true %>
+
+          <div class="js-creator-signup-username-hint-row hidden">
+            <div class="fs-xs color-base-60 flex">
+              <span>Your username will be&nbsp;</span>
+              <span class="bold color-base-80 js-creator-signup-username-hint"></span>
+              <button class="cursor-pointer border-none bg-transparent p-0 js-creator-edit-username" role="button">
+                <%= inline_svg_tag("pencil.svg", class: "cursor-pointer", aria: true, title: "Pencil icon") %>
+              </button>
```

Comment:
```suggestion
              <button aria-label="Edit username" class="cursor-pointer border-none bg-transparent p-0 js-creator-edit-username" role="button">
                <%= inline_svg_tag("pencil.svg", class: "cursor-pointer", aria_hidden: true, title: "Edit username") %>
              </button>
```

Just a suggestion to make this a bit more accessible, and change the hover text that appears over the SVG